app/controllers/users_controller.rb

```diff
@@ -6,6 +6,16 @@ def index
     @users = User.all
   end
 
+  def update
+    @user = User.find(params[:id])
+    if @user.update(user_params)
+      flash[:success] = t('user.successful_update')
+      redirect_to users_path
```

Comment:
Usually after record updating you need to redirect to record page, but not to the collection page